.grit/patterns/python/warning-for-tempfile-without-flush.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Warning for tempfile without flush
```

Comment:
Use a shorter file name. And don't call it a "warning" - the level of a diagnostic is decided by the consumer: https://docs.grit.io/guides/config#patterns